google/ggrc-core
# GGRC-4869 Handle empty Admin on Cycles generation gracefully.

# Issue description

Nightly cron job tries to start planned cycle for Workflow without Admin. It fails, cause `build_cycle` procedure starts a cycle on behalf of first Admin from Workflow. 

# Steps to test the changes

- Create Workflow with Admin.
- Delete Admin from `access_control_list` in DB using RAW SQL.
- Run nightly cron job (simulate with running `start_recurring_cycles`). I think that best way to check it, use integration tests.

# Solution description

- On attempt to start a cycle for broken Workflow added error message into log.
- Cycle generation will be skipped for Workflow without Admin.

# Sanity checklist




# PR Review checklist
